frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -117,6 +119,10 @@ export const getThemeSpecificCssVariables = (theme: MantineTheme) => css`
     .map(([cssVar, themeKey]) => {
       const value = getIn(theme.other, themeKey.split("."));
 
+      if (cssVar === "--mb-overlay-z-index") {
+        return `${cssVar}: ${value ?? OVERLAY_Z_INDEX ?? 200};`;
```

Comment:
I think the `OVERLAY_Z_INDEX` should always be 200 right?

```suggestion
        return `${cssVar}: ${value ?? OVERLAY_Z_INDEX};`;
```

Reply:
I don't know why, but Cypress couldn't find the value 200 unless I added it again here.

Reply:
I'm not sure why we need this condition here, as we could define `--mb-overlay-z-index` via CSS, and it would've been applied via the `"--mb-overlay-z-index": "popover.zIndex"` mapping above if a theme value is provided.